test/smoke-dev/launch_latency/Makefile

```
@@ -9,7 +9,7 @@ CLANG        ?= clang
 OMP_BIN      = $(AOMP)/bin/$(CLANG)
 CC           = $(OMP_BIN) $(VERBOSE)
 RUNENV       = ulimit -s unlimited;
-RUNCMD       = rocprofv3 --output-format csv --kernel-trace --stats -- ./$(TESTNAME) && python3 printLatency.py
+RUNCMD       = $(AOMPHIP)/bin/rocprofv3 --output-format csv --kernel-trace --stats -- ./$(TESTNAME) && python3 printLatency.py
```

```suggestion
RUNCMD       = $(AOMP)/bin/rocprofv3 --output-format csv --kernel-trace --stats -- ./$(TESTNAME) && python3 printLatency.py
```
I think we want to key off of `AOMP` here